readthedocs/projects/tasks.py

```diff
@@ -443,6 +439,22 @@ def run_setup(self, record=True):
 
         return True
 
+    def additional_vcs_operations(self):
+        # TODO: execute with lock
```

Comment:
I think this step should be done with the context management, the one that creates a lock